### DISCUSSION TITLE
Images instead of videos

### DISCUSSION THREAD

**Discussion by LjIA26** [talmolab/sleap#990] [Help!]
Hello, 

I was wondering if this framework could be adapted for use on 2D images instead of videos.

**Comment by talmo** [comment]
Hi @LjIA26,

Yep, SLEAP works on single images. The only thing is you'll want to collect them into a sequence format like a video or an HDF5 file. Do you mind giving us a little more info on how your data is organized? For example, do you just have a single folder full of JPEGs/PNGs? Multi-channel TIFFs?

Talmo

**Reply by LjIA26** [reply]
The images are in PNGs, I am just starting the labeling which is in a json file. They are plant images, they are quite different from each other, but they share a lot of similarities. Would the json file also work for the images then converted to videos?
